src/pages/Index.vue

```diff
@@ -19,7 +19,7 @@
         </div>
       </div>
       <div class="row justify-center items-center q-py-lg">
-        <div class="mouse">
+        <div class="mouse" v-on:click="scrollDown()">
```

Comment:
gib the mouse style: cursor: pointer so people now that they can click

Reply:
oh. and I think it's pretty common to use @click instead of the verbose form v-on:click